Plugins/Public/base_plugin/PlayerBase.cpp

```diff
@@ -453,6 +457,10 @@ void PlayerBase::Save()
 			fprintf(file, "commodity = %u, %u, %f, %u, %u, %u\n",
 				i.first, i.second.quantity, i.second.price, i.second.min_stock, i.second.max_stock, int(i.second.is_public));
 		}
+		for (auto i : available_blueprints) 
+		{
+			fprintf(file, "blueprint = %u\n", i);
```

Comment:
I'd make it so that blueprints are saved by their nicknames, so admins can more easily read/edit the file. Numeric hashes bad.